drivers/dai/intel/alh/alh.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INTEL_DAI_DRIVER_ALH_H__
+#define __INTEL_DAI_DRIVER_ALH_H__
+
+#include <stdint.h>
+#include <drivers/dai.h>
+#include "alh_map.h"
+
+#define DAI_NUM_ALH_BI_DIR_LINKS_GROUP 4
+
+#define ALH_STREAM_OFFSET 0x4
+
+#define IPC4_ALH_MAX_NUMBER_OF_GTW 16
+#define IPC4_ALH_DAI_INDEX_OFFSET 7
+
+#define ALH_GPDMA_BURST_LENGTH 4
+
+#define SET_BITS(b_hi, b_lo, x)	\
+	(((x) & ((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL)) << (b_lo))
```

Comment:
```suggestion
#define SET_BITS(b_hi, b_lo, x)	\
	(((x) & ((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL)) << (b_lo))
```

This sounds reasonable to go to `util_macro.h`

Reply:
moved to util_macro.h